servicepulse/usage.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Usage
+summary: Use ServicePulse to view and measure the usage of an NServiceBus system.
+component: ServicePulse
+reviewed: 2024-05-08
+related:
+  - nservicebus/usage
+---
+
+This page allows for viewing the endpoint usage summary of a system using NServiceBus.
+A usage report can be downloaded here at license renewal time.
+
+Usage page requires ServicePulse version 1.39 or later, and ServiceControl version 5.3 or later.
```

Comment:
This is not visible enough to the customer - gets lost on the page.
The note made it stand out.